src/hacbs/components/ImportForm/utils/__tests__/application-data.ts

```diff
@@ -0,0 +1,38 @@
+import { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
+
+export const mockApplication: K8sResourceCommon = {
+  apiVersion: 'appstudio.redhat.com/v1alpha1',
+  kind: 'Application',
+  metadata: {
+    creationTimestamp: '2022-08-09T18:52:59Z',
+    generation: 1,
+    managedFields: [
+      {
+        apiVersion: 'appstudio.redhat.com/v1alpha1',
+        fieldsType: 'FieldsV1',
+        fieldsV1: {
+          'f:spec': {
+            '.': {},
+            'f:displayName': {},
+          },
+        },
+        manager: 'Mozilla',
+        operation: 'Update',
+        time: '2022-08-09T18:52:59Z',
+      },
+    ],
+    name: 'my-application-3',
+    namespace: 'karthik-jk',
```

Comment:
```suggestion
    namespace: 'test',
```

Reply:
moved to __data__ folder and udpated it.